src/GPLikelihoods.jl

```diff
@@ -14,6 +14,7 @@ export BernoulliLikelihood,
     PoissonLikelihood,
     ExponentialLikelihood,
     GammaLikelihood
+export link
```

Comment:
Maybe not necessary to export it? I guess it's mainly useful internally.

```suggestion
```

Reply:
Yup I realized this later